hedera-node/hedera-app/src/main/java/com/hedera/node/app/roster/schemas/RosterTransplantSchema.java

```diff
@@ -1,16 +1,38 @@
-// SPDX-License-Identifier: Apache-2.0
+/*
```

Comment:
A couple of these files have this old header in them. Intentional?

Reply:
As I understad, we're only going to switch to the SPDX header later.